### PR TITLE
Battle status

### PR DESCRIPTION
**Pull request by bfcferreira**
Battle Status Menu fully implemented.

### CODE REVIEW

File: base/Window.ts [hunk 1]
```diff
@@ -94,6 +108,90 @@ export class Window {
         return this.group.y;
     }
 
+    make_item_obj(
+        key_name: string,
+        pos?: {x: number; y: number},
+        params?: {bg?: boolean; equipped?: boolean; broken?: boolean; quantity?: number; internal_group?: string}
+    ) {
+        let obj: ItemObj = {icon: null, background: null, equipped: null, broken: null, quantity: null};
```
Comment (jjppof) [inline base/Window.ts:116]: This var could be const.

File: base/Window.ts [hunk 2]
```diff
@@ -94,6 +108,90 @@ export class Window {
...
+        const base_x = pos ? pos.x : 0;
+        const base_y = pos ? pos.y : 0;
+
+        /*
+        this.window.make_item_obj("spirit_ring", {x: 100, y:100}, {broken: true, equipped: true});
+        
+        let obj = this.window.make_item_obj("herb", undefined, {bg: true, quantity:10});
+        this.window.move_item_obj(obj, {x: 50, y:50});*/
```
Comment (jjppof) [inline base/Window.ts:124]: Please remove unused code.

**Comment by jjppof** [comment]
This window is not showing the djinn status. Do you want to make it in this PR?
image

**Comment by bfcferreira** [comment]
> 
> 
> This window is not showing the djinn status. Do you want to make it in this PR?
> image

Resolved in the last commit.

**Review by jjppof — APPROVED** [review]

**Pull request merged**